tests/behaviour/concept/type/thingtype/thing_type_steps.py

```diff
@@ -51,7 +51,7 @@ def step_impl(context: Context, root_label: RootLabel, type_label: str):
     context.get_thing_type(root_label, type_label).as_remote(context.tx()).delete()
 
 
-@step("{root_label:RootLabel}({type_label}) is null: {is_null}")
+@step("{root_label:RootLabel}({type_label:Label}) is null: {is_null}")
```

Comment:
Here and on line 65 disambiguate the rules with new ones on ines 346 and 353 respectively. Without this change `{type_label}` matched phrases like `"customer) get owns overridden attribute(work-email"`, which is nonsense.

Reply:
NB: there is a FIXME in parameters.py about consistently using type hints in the rules here, but I'd argue this is out of scope for this PR.